config/branch.go

```diff
@@ -54,6 +49,10 @@ func (b *Branch) Validate() error {
 		return errBranchInvalidRebase
 	}
 
+	// Skip validation on empty branch names to avoid violating git ref format rules
+	if b.Name == "" {
+		return nil
+	}
```

Comment:
Not sure we should change the existing branch validation, name seems to be compulsory here.